management/models.py

```diff
@@ -29,10 +37,13 @@ def set_permissions(self):
         """Set object-level permissions."""
 
         delete, change, view = _get_perm_codenames(self.__class__)
+        standard_group = Group.objects.get(name="Standard")
+        anonymous_user = get_anonymous_user()
 
-        # Assign view permissions for all users
+        # View permissions based on permissions level
         if self.permissions_level in ["public", "internal"]:
-            assign_perm(view, User.objects.all(), self)
+            assign_perm(view, standard_group, self)
+            assign_perm(view, anonymous_user, self)
```

Comment:
If it is internal, the anonymous user should not have `view` permissions, right?

Reply:
So this is confusing. I've interpreted the internal status to relate to the view_measurements permission only. i.e. if a station is internal, the measurements are only visible to logged-in users, but the station metadata (which this permission relates to) is visible to everyone. Does this seem appropriate?

Reply:
True. I forgot about the discussion we had the other day. Just to be sure, @ICHydro , is this the right interpretation?